### ISSUE TITLE
Bug : app instantly crashes on open

### ISSUE THREAD

**Issue by s16899** [crash5band/MikuMikuWorld#65]
So I opened mikumikuworld latest release, and instantly, the app crashes. Anyone got the same issues?

**Comment by crash5band** [comment]
Can you provide more information like your PC specs and operating system?

**Comment by s16899** [comment]
I'm using Windows 11 with AMD Radeon(TM) Graphics (built-in GPU for asus laptops) and AMD Ryzen 5 5000 series

**Comment by crash5band** [comment]
That's weird. Do you even get any error messages or does it just not launch at all?

**Comment by XiaoThewind** [comment]
I have the same issue, and i don't know how to fix it.

**Comment by Amia-Yuki27** [comment]
I also got crash, when i extracted it and clicking on it, it never opened and always crashed. please i don't know to fix it...

**Comment by crash5band** [comment]
Are you getting any sort of error message?

**Comment by Amia-Yuki27** [comment]
> Are you getting any sort of error message?

Oh, never mind, I fixed it by myself yesterday. But thank you for asking

**Comment by crash5band** [comment]
> Oh, never mind, I fixed it by myself yesterday. But thank you for asking
If you don't mind, please post how you fixed it here since it might help other people experiencing crashes on opening

**Comment by Haliolo** [comment]
yes, please tell how to fix this, because i am having the same problem

**Comment by starbyy** [comment]
FIX: you dont have visual c++ redistributable.

**Comment by s16899** [comment]
> FIX: you dont have visual c++ redistributable.

Oh, that’s kinda weird, I’ll try downloading that.
